data/src/main/java/com/globant/data/service/response/PokemonSupertypesResponse.kt

```diff
@@ -0,0 +1,5 @@
+package com.globant.data.service.response
+
+class PokemonSupertypesResponse<T>(
```

Comment:
Why do you need a generic here? The response isn't String?

Reply:
that's true, changed the generic for the actual response type